CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.2.1
```

Comment:
I think this is a new feature and it is reasonable to update your pubspec to require it. This should be 1.3.0

Reply:
Hmm, I was thinking of it as a bug fix, but I could see it either way (fixing the bug does enable the use of symlinks which could be considered a feature?)